### PR TITLE
feat: remove user enrollment on event

### PR DESCRIPTION
**Pull request by rubuy-74**
Admins are now able to remove user's enrollment for a given event.

Careful, that can be dangerous.

### CODE REVIEW

File: app/Http/Controllers/EventController.php [hunk 1]
```diff
@@ -99,6 +101,33 @@ public function join(Request $request, Event $event)
         return redirect()->route('profile.show')->banner('Inscrição realizada com sucesso!');
     }
 
+    public function remove(Request $request,Event $event, Enrollment $enrollment)
```
Comment (limwa) [inline app/Http/Controllers/EventController.php:104]: You need authorization for this endpoint, or else any user can remove the enrollment of other users. I think there is a middleware to enforce that the calling user is an admin

**Review by limwa — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/EventController.php (hunk 1 above)
Comment (limwa) [reply app/Http/Controllers/EventController.php:104]: There is no middleware, but you can use something like this to achieve the same effect https://github.com/semanadeinformatica/website-24/blob/93ed09e7916c74aaa8e3397788a67169d5e42107/app/Http/Controllers/EventController.php#L37

Comment (rubuy-74) [reply app/Http/Controllers/EventController.php:104]: Yeah, I added a check for users that are admin, being logged in or not. I think its kinda dumb, but a good enough solution for now.

**Review by limwa — APPROVED** [review]

File: app/Http/Controllers/EventController.php [hunk 2]
```diff
@@ -103,29 +103,30 @@ public function join(Request $request, Event $event)
 
     public function remove(Request $request,Event $event, Enrollment $enrollment)
     {
-        $edition = $request->input('edition');
+        if (Gate::allows('admin') || Gate::allows('staff', [$edition])) {
```
Comment (limwa) [inline app/Http/Controllers/EventController.php]: this is useless, considering that the check was added at the middleware level

Comment (limwa) [reply app/Http/Controllers/EventController.php]: this is different than the middleware though, the `can:admin` middleware only enforces `Gate::allows("admin")`

**Review by limwa — APPROVED** [review]
LGTM

**Pull request merged**